packages/form-layout/src/vaadin-form-layout-mixin.js

```diff
@@ -259,6 +273,14 @@ export const FormLayoutMixin = (superClass) =>
       return [`${formLayoutSlotStyles}`.replace('vaadin-form-layout', this.localName)];
     }
 
+    /**
+     * @override
+     * @protected
+     */
+    get slotStyles() {
+      return [`${formLayoutSlotStyles}`.replace('vaadin-form-layout', this.localName)];
+    }
```

Comment:
We already have this getter a few lines above.

Reply:
Thanks! That slipped through during the rebase. Removed.